pom.xml

```diff
@@ -81,17 +79,17 @@
 		<repository>
 			<id>spring-maven-release</id>
 			<name>Spring Maven Release Repository</name>
-			<url>https://maven.springframework.org/release</url>
+			<url>http://maven.springframework.org/release</url>
```

Comment:
Any reason these urls have been downgraded from the https protocol?

Reply:
upgraded